docs/design/specs/light-mode.md

```diff
@@ -62,11 +62,13 @@ This procedure can be visualized in a flow chart:
 :::{.mermaid-container}
 ```mermaid
 flowchart TB
-  start([begin]) --> init[j := 0]
+  start([begin]) --> init[j := 0\ng := 0\ngap := 20]
   init --> query[query addr_j]
   query --> tx{transactions?}
-  tx -- no --> g{happened g times?}
-  tx -- yes --> add[j := j+1]
+  tx -- no --> gapAdd[g := g+1] 
+  gapAdd --> g{g > gap?}
```

Comment:
`>=` or `>`? 😅 The rule is "Between any two used addresses, there are at most `gap` many consecutive unused addresses. (An address is used if the query returns a nonempty list of transactions, unused otherwise.)".

Reply:
This seems correct to me: `g` counts unused addresses, if `g` is greater than `gap` then search is over, thus allowing it to have at most `g = gap` unused addresses.

Reply:
> `g` counts unused addresses

That's a very good argument / invariant.